src/components/FormInput/FormInput.tsx

```diff
@@ -9,16 +9,9 @@ interface FormInputProps {
 }
 
 const FormInput: React.FunctionComponent<FormInputProps> = (props: FormInputProps) => {
-  const { placeholder, value, onChangeText } = props;
+  const { placeholder, onChangeText } = props;
 
-  return (
-    <TextInput
-      style={styles.input}
-      placeholder={placeholder}
-      value={value}
-      onChangeText={onChangeText}
-    />
-  );
+  return <TextInput style={styles.input} placeholder={placeholder} onChangeText={onChangeText} />;
```

Comment:
where specifically is this used?

Reply:
in the phone number and verification code screens

Reply:
oh I see. if it's not a generic component for forms then can we name the component to be more explicit i.e. `PhoneNumberInput`?